pkg/render/logstorage_test.go

```diff
@@ -617,6 +617,105 @@ var _ = Describe("Elasticsearch rendering tests", func() {
 			oidcUserSecret := GetResource(createResources, render.OIDCUsersEsSecreteName, render.ElasticsearchNamespace, "", "v1", "Secret")
 			Expect(oidcUserSecret).ShouldNot(BeNil())
 		})
+
+		Context("ECKOperator memory requests/limits", func() {
+			When("LogStorage Spec contains an entry for ECKOperator in ComponentResources", func() {
+				It("should set matching memory requests/limits in the elastic-operator StatefulSet Spec, manager container", func() {
```

Comment:
I think this test should not include all those other resource checks. I would prefer it to be focused on the minimum test necessary to see that the memory requests/limits are set, because with all this other stuff then if any of them change or are removed this test which is unrelated has to be updated.